source/integrate-with-integration-environment/authenticate-your-user.html.md.erb

```diff
@@ -193,6 +193,19 @@ If your request included the `state` parameter, the URI will also include this p
 
 ### Error handling for ‘Make a request to the `/authorize` endpoint’
 
+You must check the HTTP return code from   the `/authorize` request.
+
+#### HTTP 502 Bad Gateway
+If you receive `{"message": "Internal server error"}` with an HTTP 502 (Bad Gateway) in response to your response `GET` request to the `/authorize` endpoint you should check the [parameters are correct][integrate.replace-example-message].
+
+Common reasons for the error include:
+
+* the `client_id` parameter does not match your client configuration
+* the `redirect_uri` parameter does not match one of the uris configured in your client configuration, <%= warning_text('this is a common cause of the error and it is important that the match is exact and includes the protocol (`https://` or `http`) as described by <a href="https://openid.net/specs/openid-connect-core-1_0.html#AuthRequest">OpenID Connect Core 1.0</a>') %>
+* the `response_type` parameter is not set to `code`
+* `openid` is missing from the `scope` parameter. If `openid` is present but other incorrect scopes are also present then the `invalid_scope` is returned with an HTTP 302 instead. 
```

Comment:
```suggestion
* `openid` is missing from the `scope` parameter. If `openid` is present but other incorrect scopes are also present then you will receive an HTTP 302 error with `invalid_scope` instead. 
```

Does that keep the same meaning? I just think it wasn't clear where the `invalid_scope` came from but if it's with that error that makes sense to me. Feel free to disagree if it's not right though!

Reply:
```suggestion
* `openid` is missing from the `scope` parameter. If `openid` is requested but other incorrect scopes are also requested the error `invalid_scope` is returned with an HTTP 302 instead. 
```